scripts/guest/m-reinstall

```diff
@@ -8,6 +8,7 @@ get_config_value="/vagrant/scripts/get_config_value.sh"
 magento_host_name="$(bash ${get_config_value} "magento_host_name")"
 use_nfs="$(bash "${vagrant_dir}/scripts/get_config_value.sh" "guest_use_nfs")"
 is_windows_host=${IS_WINDOWS_HOST}
+generate_data="$(bash ${get_config_value} "magento_generate_sample_data")"
```

Comment:
The option is not defined in `config.yaml` by default.
Please make sure that sample data is installed by default, including those who have an instance already installed.